src/arrayMethodPop.js

```diff
@@ -6,7 +6,10 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    let result = +this.splice([this.length - 1], 1);
+    if (result > 0) {
+      return result;
+    } else return undefined;
```

Comment:
return undefined is not really needed. This code will do the same without 'return undefined'.